doc/source/ray-more-libs/lightgbm-ray.md

```diff
@@ -0,0 +1,8 @@
+<!--
```

Comment:
do we need one for xgboost as well?

Reply:
There is a file there for xgboost already